docs/reference/flowdecomposition.rst

```diff
@@ -0,0 +1,18 @@
+Flow decomposition
+==================
+
+.. module:: pypowsybl.flowdecomposition
+
+The flow decomposition module allows to decompose flows on cross-border relevant network element with contingency (XNEC) based on the ACER methodology.
+This python interface is based on the java implementation in the powsybl ENTSO-E repository.
```

Comment:
Add a link to the java implementation if you can.

Reply:
done